RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -678,10 +717,36 @@ public static String getUsedPluginVersion(String pluginID) {
         return LibrariesFactory.PLUGIN_VERSION_CURRENT;
     }
 
+    public static ShellCommandSet getLocalCommandSet() {
+        String localCommandSet = getApplicationProperty("localCommandSet");
+
+        // Handle default case, the local commandset ist not set.
+        if (RoddyConversionHelperMethods.isNullOrEmpty(localCommandSet)) {
+            logger.postSometimesInfo("Using BashCommandSet, localCommandSet ist not set.");
+            BashCommandSet bcs = new BashCommandSet();
+            bcs.validate();
+            return bcs;
+        }
+
+        Class<ShellCommandSet> cls = null;
+        try {
+            cls = (Class<ShellCommandSet>) LibrariesFactory.getGroovyClassLoader().loadClass(localCommandSet);
+            ShellCommandSet shellCommandSet = cls.newInstance();
+            if (shellCommandSet.validate())
+                return shellCommandSet;
+            throw new RuntimeException("The selected ShellCommandSet '${localCommandSet}' could not validate.");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Could not find localCommandSet class '${localCommandSet}'. Check you ini file.");
```

Comment:
This is also something, where you don't want to show a call stack. The user has apparently a typo or some fantasy value in there. So maybe again: descriptive message and exit directly.